client/i18n/translations-schema.js

```diff
@@ -445,7 +447,8 @@ const translationsSchema = {
     'update-email-2': 'Update your email address here:',
     email: 'Email',
     and: 'and',
-    'change-theme': 'Sign in to change theme.'
+    'change-theme': 'Sign in to change theme.',
+    'translation-pending': 'Help us translate.'
```

Comment:
```suggestion
    'translation-pending': 'Help us translate'
```

@ahmadabdolsaheb you are correct. Fixing this.